cockatrice/src/tab_deck_editor.cpp

```diff
@@ -863,6 +873,43 @@ void TabDeckEditor::actOpenCustomsetsFolder() {
 
 }
 
+void TabDeckEditor::actAddCustomSet()
+{
+
+#if QT_VERSION < 0x050000
+    QString dataDir = QDesktopServices::storageLocation(QDesktopServices::DataLocation);
+#else
+    QString dataDir = QStandardPaths::standardLocations(QStandardPaths::DataLocation).first();
+#endif
+
+    if (!confirmClose())
+        return;
+
+    QFileDialog dialog(this, tr("Load sets/cards"));
+    dialog.setDirectory(dataDir);
+    dialog.setNameFilters(TabDeckEditor::fileNameFilters);
+    if (!dialog.exec())
+        return;
+
+    QString fileName = dialog.selectedFiles().at(0);
+    QDir dir(dataDir.append("/customsets"));
+    QStringList files = dir.entryList();
+    int maxIndex = 0;
+    for (int i = 0; i < files.size(); ++i) {
+        int fileIndex = files.at(i).split(".").at(0).toInt();
```

Comment:
This grabs the next index for custom sets, which must start with a number code, zero-filled to two digits, at least according to the wiki, will make suggested changes.